python/lsst/pipe/supertask/activator.py

```diff
@@ -392,6 +428,10 @@ def parse_and_run(cls):
                                       help="Manually adds packages to look for task, overrides "
                                            "TASK_PACKAGES, to see the available packages use -lp "
                                            "option")
+        parser_activator.add_argument('--package_paths', nargs='+', default=(),
+                                      help="Manually adds path of packages to look for task, overrides "
+                                           "TASK_PACKAGES, to see the available packages use -lp "
+                                           "option")
```

Comment:
So a full path to a .py file? Relative paths can be used relative to where the command line is started?